psc-ide.el

```diff
@@ -120,9 +136,50 @@
 ;;
 ;; Non-interactive.
 
+(defun psc-ide-parse-exposed (exposed)
+  "Parsed the EXPOSED names from a qualified import."
+  (if exposed
+      (mapcar (lambda (item)
+                (s-trim item))
+              (s-split "," exposed))
+    nil))
+
+(defun psc-ide-extract-import-from-match-data (&optional string)
+
+  "Helper function for use when using the psc-ide-import-regex to match imports to extract
+the relevant info from the groups. STRING is for use when the search used was with `string-match'."
+
+  (let* ((data (match-data))
+         (len (length data))
+         (idx 3)
+         result)
+    (push `(module . ,(match-string-no-properties 1 string)) result)
+    (push `(alias . ,(match-string-no-properties 2 string)) result)
+    (push `(exposing . ,(psc-ide-parse-exposed (match-string-no-properties 3 string))) result)
+    result))
+
+(defun psc-ide-parse-imports-in-buffer (&optional buffer)
+
+  "Parse the list of imports for the current elm BUFFER."
```

Comment:
Haha thanks for spotting that.  I had started writing a company-elm plugin a while back that never got finished but reused some pieces of that code.  Should have did a search and replace =)